src/background/services/paymentSession.ts

```diff
@@ -161,15 +165,20 @@ export class PaymentSession {
         } else if (isTokenExpiredError(e)) {
           await this.openPaymentsService.rotateToken()
           continue
+        } else if (isOutOfBalanceError(e)) {
+          const switched = await this.openPaymentsService.switchGrant()
+          if (switched === null) {
+            this.events.emit('open_payments.out_of_funds')
+          } else {
+            await this.setIncomingPaymentUrl()
+          }
```

Comment:
Why are we creating a new incoming payment?

Reply:
Better context with lines:
https://github.com/interledger/web-monetization-extension/blob/7e318b8937dc3065707c42f7e2e33ac796ff88a3/src/background/services/paymentSession.ts#L168-L175

Reply:
Oh, I saw accessToken used there and thought it was the one from grant we created. Will fix.